Services/Services_feeds/reddit_service_feed/reddit_feed.py

```diff
@@ -151,3 +147,11 @@ async def _start_service_feed(self):
                 self.keep_running = False
                 self.should_stop = True
         return False
+
+    async def _start_service_feed(self):
+        self.listener_task = asyncio.create_task(self._start_listener_task())
+        return True
+
+    async def stop(self):
+        await super().stop()
+        self.listener_task.cancel()
```

Comment:
Should we catch asyncio cancelError here ?

Reply:
No need for it as the task is never awaited